Bug: WinUI projects have been built with a corrupt build of WebView2

### Description

Looks like some of the WinUI projects have been built with a corrupt build of WebView2 that comes with WinAppSDK.

### Steps to reproduce

1. Install a WinUI Toolkit project (`Riverside.Toolkit.WinUI.Helpers` is what I've tested)
2. Attempt to build the project that references it
3. See an error in the NuGet restore process

### Version

v2.0.0-beta1 - v2.0.0-beta3

### Affected platforms

_No response_

### Platform version

(any)

### Log File

```b
error MSB3030: Could not copy the file "%userprofile%\.nuget\packages\riverside.toolkit.winui.helpers\2.0.0-beta1\lib\net9.0-windows10.0.26100\Riverside.Toolkit.WinUI.Helpers\runtimes\win-arm64\native\WebView2Loader.dll" because it was not found.
```

### Comments

> I think that package is busted
> It was likely built with the old WebView2 build with that bug
> So they embedded that .dll reference into the PRI file
> You need them to build a new version with thew new package

--- a/eng/Tooling.props
+++ b/eng/Tooling.props
@@ -40,6 +40,7 @@
 		<IsAotCompatible>true</IsAotCompatible>
 	</PropertyGroup>
 	<ItemGroup Condition="'$(IsWinUIProject)' == 'true'">
+		<PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2957.106" />
 		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.26100.1742" />
 		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.241114003" />
 		<PackageReference Include="WinUIEx" Version="2.5.1" />
@@ -62,6 +63,7 @@
 		<IsAotCompatible>true</IsAotCompatible>
 	</PropertyGroup>
 	<ItemGroup Condition="'$(IsUwpProject)' == 'true' ">
+		<PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2957.106" />
 		<PackageReference Include="Microsoft.UI.Xaml" Version="2.8.6" />
 		<PackageReference Include="Win2D.uwp" Version="1.28.1" />
 		<PackageReference Include="DependencyPropertyGenerator" Version="1.4.0">

